.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,6 +1,7 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0" is_locked="false">
     <option name="myName" value="Project Default" />
+    <inspection_tool class="35f0afe5-6749-3bcd-88fe-638b65ec3b43" enabled="true" level="ERROR" enabled_by_default="true" editorAttributes="ERRORS_ATTRIBUTES" />
```

Comment:
Do we want this line?

Reply:
I don't understand that line, but it's what makes this an error instead of a warning.